src/Components/Artwork/GridItem.tsx

```diff
@@ -48,14 +48,24 @@ class ArtworkGridItemContainer extends React.Component<Props, null> {
           <a href={artwork.href}>
             <Image src={artwork.image.url} />
           </a>
-          <SaveButtonBlock
-            className="artwork-save"
-            artwork={artwork as any}
-            style={{ position: "absolute", right: "10px", bottom: "10px" }}
-            useRelay={useRelay}
-            {...currentUserSpread}
-            mediator={this.props.mediator}
-          />
+          <Responsive>
+            {({ hover }) =>
+              hover && (
```

Comment:
Here is an example using the `Responsive` component…

Reply:
This seems to...not work? The artwork grids present on the gene page (for example), on desktop web, are now lacking save buttons. I feel like I may have missed some of the nuance in this PR, was this an oversight or am I missing something else?